hypertrace-trace-enricher-impl/src/test/java/org/hypertrace/traceenricher/enrichment/enrichers/resolver/backend/BackendEntityResolverTest.java

```diff
@@ -634,9 +632,7 @@ public void checkFallbackBackendEntityGeneratedFromClientExitSpan() {
         .setEventId(ByteBuffer.wrap("random".getBytes()))
         .setAttributes(Attributes.newBuilder().setAttributeMap(Map
             .of(
-                "span.kind", AttributeValue.newBuilder().setValue("server").build(),
-                JAEGER_SERVICE_NAME_ATTR_NAME,
-                AttributeValue.newBuilder().setValue("customer").build())).build())
+                "span.kind", AttributeValue.newBuilder().setValue("server").build())).build())
```

Comment:
is there is not need of setting `.setServiceName("customer")` here?

Reply:
No, it was redundant, this test if for fallback on ```ClientSpanEndpointResolver``` , in this class, parent service name is extracted off enriched attributes(being set in the line below)